test/autotest.py

```diff
@@ -556,8 +556,10 @@ def testRestart():
       printFixed("ckpt:")
       # NOTE:  If this faile, it will throw an exception to CheckFailed
       #  of this function:  testRestart
+      sleep(S*SLOW)
       testCheckpoint()
       printFixed("PASSED ")
+      sleep(S*SLOW)
```

Comment:
If `testCheckpoint()` has indeed succeeded (evident from "PASSED" above), then what's the point of doing a sleep here?